### PR TITLE
Add content to usage page

### PR DESCRIPTION
**Pull request by ka0ula**

### CODE REVIEW

File: docs/getting_started/usage.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+- **Excessive use consequences**: Regularly exceeding the limit may result in:
+  
+  - Notifications about potential performance degradation.
+  - Adjusted fees for exceeding the limit.
+  - Recommendations for upgrading to plans with higher allowances.
+
+Tracking consumption empowers users to proactively adjust their data usage and consider upgrading their plans if necessary.
+
+!!! note
+    If you are on an **Intelligence pla**n only, please note that you will not see any analyzed bytes, as there are none associated with this plan.
```
Comment (Sengthay) [inline docs/getting_started/usage.md:46]: pla**n

File: docs/getting_started/usage.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Usage overview
+
+The Usage page provides users with a comprehensive overview of their data consumption and performance metrics associated with their communities. This feature is essential for understanding resource allocation and ensuring compliance with the fair use policy. Users can access detailed analytics regarding their analyzed bytes by community, as well as all executed playbook actions, facilitating efficient monitoring of usage and prevention of service degradation.
```
Comment (gbossert) [inline docs/getting_started/usage.md:3]:
```suggestion
The Usage page provides users with a comprehensive overview of their data consumption and performance metrics associated with their communities. This feature is essential for understanding resource allocation and ensuring compliance with the usage policy. Users can access detailed analytics regarding their analyzed bytes by community, as well as all executed playbook actions, facilitating efficient monitoring of usage and prevention of service degradation.
```

File: docs/getting_started/usage.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## How it works
+
+Upon accessing the Usage page, you are presented with a dashboard that displays their data usage for the current month. The dashboard consists of key sections:
```
Comment (gbossert) [inline docs/getting_started/usage.md:7]:
```suggestion
Upon accessing the Usage page, you are presented with a dashboard that displays current month data usage. The dashboard consists of key sections:
```

File: docs/getting_started/usage.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+- **Analyzed Bytes**: Visualizes the total data analyzed by the community and indicates whether the usage is within acceptable limits.
```
Comment (gbossert) [inline docs/getting_started/usage.md:9]:
```suggestion
- **Analyzed Bytes**: Visualizes the total data analyzed by the community.
```

File: docs/getting_started/usage.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+- **Type of subscription**: The subscription plan type.
+- **Analyzed bytes**: The amount of data analyzed, applicable to your plan.
+
+You can also have a look at your previous consumption by clicking on the date selector in the upper right of the screen. 
+
+## Sections
+
+### Analyzed bytes
+
+In the Analyzed bytes section, users can view a detailed breakdown of their monthly data usage in relation to the Fair Use Policy.
```
Comment (gbossert) [inline docs/getting_started/usage.md:25]:
```suggestion
In the Analyzed bytes section, users can view a detailed breakdown of their monthly data usage in relation to their usage policy.
```

File: docs/getting_started/usage.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+- **Current usage**: Displays the total bytes analyzed (e.g., 100 GB) and highlights any excess usage against the predefined fair use limit (e.g., 210 GB).
```
Comment (gbossert) [inline docs/getting_started/usage.md:27]:
```suggestion
- **Current usage**: Displays the total bytes analyzed (e.g., 100 GB).
```

File: docs/getting_started/usage.md (hunk 1 above)
Comment (gbossert) [inline docs/getting_started/usage.md:46]:
```suggestion
    If you are on an **Intelligence plan** only, please note that you will not see any analyzed bytes, as there are none associated with this plan.
```

File: docs/getting_started/usage.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+Tracking consumption empowers users to proactively adjust their data usage and consider upgrading their plans if necessary.
+
+!!! note
+    If you are on an **Intelligence pla**n only, please note that you will not see any analyzed bytes, as there are none associated with this plan.
+
+### Playbook Actions
+
+The Executed Playbook Actions section displays the total number of actions completed within the selected month. This is calculated by:
+
+- **Number of actions**: Each automated task initiated by the user within their playbooks is logged. The total actions executed by the user are aggregated and presented accordingly.
```
Comment (gbossert) [inline docs/getting_started/usage.md:52]:
```suggestion
- **Number of actions**: Each playbook action initiated by the user within their playbooks is logged. The total actions executed by the user are aggregated and presented accordingly.
```

File: docs/getting_started/usage.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+Depending on the plan you are subscribed to, you may have different thresholds for data usage:
+
+- **Defend CORE**: Fair use threshold is set at **3GB per asset per month**.
+- **Defend PRIME**: Higher threshold allowing for **5GB per asset per month**.
+
+Both plans implement a soft violation policy. If a user exceeds the set thresholds, they will be notified, but no immediate penalties will be enforced.
+
+### Defend FLOW Plan
+
+The FLOW plan establishes threshold metrics for data usage based on a maximum of **5GB per asset per month**.
```
Comment (gbossert) [inline docs/getting_started/usage.md:72]:
```suggestion
The FLOW plan establishes threshold metrics for data usage based on a predefined number of GB per month. 
```

File: docs/getting_started/usage.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+- **Defend CORE**: Fair use threshold is set at **3GB per asset per month**.
+- **Defend PRIME**: Higher threshold allowing for **5GB per asset per month**.
+
+Both plans implement a soft violation policy. If a user exceeds the set thresholds, they will be notified, but no immediate penalties will be enforced.
+
+### Defend FLOW Plan
+
+The FLOW plan establishes threshold metrics for data usage based on a maximum of **5GB per asset per month**. 
+
+#### Soft and hard limits
```
Comment (gbossert) [inline docs/getting_started/usage.md:74]: This section should be removed

File: docs/getting_started/usage.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+An asset refers to any equipment that produces logs, including user workstations, servers, and virtual machines. Customers subscribe based on the number of assets they wish to monitor, which directly influences their data usage and associated costs.
+
+## Plans and usage
+
+### Defend CORE and PRIME plans
+
+Depending on the plan you are subscribed to, you may have different thresholds for data usage:
+
+- **Defend CORE**: Fair use threshold is set at **3GB per asset per month**.
+- **Defend PRIME**: Higher threshold allowing for **5GB per asset per month**.
```
Comment (gbossert) [inline docs/getting_started/usage.md:66]:
```suggestion
- **Defend PRIME**: Faire use threshold is set at **5GB per asset per month**.
```

File: docs/getting_started/usage.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+## Sections
+
+### Analyzed bytes
+
+In the Analyzed bytes section, users can view a detailed breakdown of their monthly data usage in relation to the Fair Use Policy.
+
+- **Current usage**: Displays the total bytes analyzed (e.g., 100 GB) and highlights any excess usage against the predefined fair use limit (e.g., 210 GB).
+- **Fair use indicator**: A visual marker shows whether current consumption exceeds the fair use threshold.
+
+### Fair use policy
```
Comment (gbossert) [inline docs/getting_started/usage.md:30]: Not sure we should document our usage policies in this document. It should be in the contract.

Comment (ka0ula) [reply docs/getting_started/usage.md:30]: @gbossert you recommend removing the whole section? I think we need to explain what we mean by fair use and how it is calculated

**Review by Sengthay — APPROVED** [review]

**Pull request merged**